ios/FastImage/FFFastImageViewManager.m

```diff
@@ -33,5 +34,13 @@ - (FFFastImageView*)view {
     [[SDWebImagePrefetcher sharedImagePrefetcher] prefetchURLs:urls];
 }
 
+RCT_EXPORT_METHOD(deleteImage:(NSString *)url)
+{
+    NSURL *nsurl = [NSURL URLWithString:url];
+    SDWebImageManager *imageManager = [SDWebImageManager sharedManager];
+    NSString *cacheKey = [imageManager cacheKeyForURL:nsurl];
+    [[SDImageCache sharedImageCache] removeImageForKey:cacheKey fromDisk:TRUE withCompletion:0];
+}
```

Comment:
Please note that SDWebImage's image cache have Async version API to delete the cache. And the JavaScript API also using Promise. So the better and robust code of this can be:

```objective-c
RCT_EXPORT_METHOD(deleteImage:(NSString *)url
                     resolver:(RCTPromiseResolveBlock)resolve
                     rejecter:(RCTPromiseRejectBlock)reject)
{
    NSURL *nsurl = [NSURL URLWithString:url];
    SDWebImageManager *imageManager = [SDWebImageManager sharedManager];
    NSString *cacheKey = [imageManager cacheKeyForURL:nsurl];
    NSString *cachePath = [[SDImageCache sharedImageCache] cachePathForKey:cacheKey];
    [[SDImageCache sharedImageCache] removeImageForKey:cacheKey fromDisk:YES withCompletion:^{
        resolve(cachePath);
    }];
}
```

Note: SDWebImage on iOS, does not populate the Error during deleting cache, since it's trivial for most users. But if you really want the error info, you need to use the `NSFileManager` and delegate for SDImageCache. But it's little hard to do so (You need to keep a alive instance and pass the context from React to keep the error info)

Reply:
Acutally, does the promise fulfilled value `cachePath` means anything ? What if the cache is in memory only but not disk (the `cachePath` will be nil, but promise is still fulfilled)